src/main/resources/com/cloudbees/simplediskusage/QuickDiskUsagePlugin/refresh.js

```diff
@@ -0,0 +1,10 @@
+function refresh(link) {
+
+    fetch(link.href, {
+        method: "POST",
+        headers: crumb.wrap({})
+    }).catch(() => {});
+
+    hoverNotification('${%Refresh scheduled}', link);
```

Comment:
Second argument here used to be `a.parentNode`, why are we changing this?

Reply:
Parent is just a <span> so the behavior is identical. But I'll keep as it was before for consistency.